openmp/runtime/test/misc_bugs/omp__kmpc_fork_call_if.c

```diff
@@ -0,0 +1,22 @@
+// RUN: %libomp-compile  -Wno-implicit-function-declaration && %t | FileCheck %s
```

Comment:
Why do we need `-Wno-implicit-function-declaration`?

Reply:
I think LLVM has moved to C99 standards recently because of which I was getting `error: call to undeclared function '__kmpc_fork_call_if'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]` so i've added this flag suppresses this error.

Reply:
Then you need to add declaration of `__kmpc_fork_call_if`

Reply:
I had tried to declare `void __kmpc_fork_call_if(ident_t *loc, kmp_int32 argc, kmpc_micro microtask, kmp_int32 cond, void *args)` but it contains certain types which are not being found.

Reply:
You can define those types on your own. See `openmp/runtime/test/atomic/kmp_atomic_cas.c` for more details. Essentially you will just need `ident_t`.

Reply:
Thank you for sharing your insights, I've defined required types and removed `-Wno-implicit-function-declaration`